### ISSUE TITLE
Siren in the vehicle not outside.

### ISSUE THREAD

**Issue by DatedGore** [photonle/Photon#118]
## Expected Behaviour
The siren to be louder outside the vehicle not inside.

## Actual Behaviour
The siren is louder inside the vehicle not outside.

## Steps to Reproduce

  1.  Have a siren
  2. Go inside.
  3. Turn on siren
  4. Go to first person
  5. Listen
  6. Go to third person with siren on
  7. Listen thats its quieter outside then inside.

## Details
- Source Github and Addon

**Comment by JoshPiper** [comment]
I don't think this is possible to fix, just because of the way Source sound works.
